File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Agent_Update.ascx.cs
using NotificationSystem.NotificationSystem.Data.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_Agent_Update : System.Web.UI.UserControl
	{
		protected void Page_Load(object sender, EventArgs e)
		{
            clsNotificationSystem theNotificationSystem = new clsNotificationSystem();

            if (!(Page.IsPostBack))
            {
                cmbAgent.DataSource = theNotificationSystem.GetAgents();
                cmbAgent.DataTextField = "AgentID"; cmbAgent.DataValueField = "FirstN";
                cmbAgent.DataBind();
            }
        }
        public void ClearControl()
        {
            this.cmbAgent.Value = string.Empty; cmbAgent.Text = "--";
        }
    }
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Agent_Update.ascx.vb
Imports System
Imports System.Web.UI
Imports NotificationSystem.NotificationSystem.Data.Classes

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Agent_Update
        Inherits UserControl
        Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
            Dim theNotificationSystem As New clsNotificationSystem

            If Not MyBase.Page.IsPostBack Then
                cmbAgent.DataSource = theNotificationSystem.GetAgents()
                cmbAgent.DataTextField = "AgentID"
                cmbAgent.DataValueField = "FirstN"
                cmbAgent.DataBind()
            End If
        End Sub
        Public Sub ClearControl()
            cmbAgent.Value = String.Empty
            cmbAgent.Text = "--"
        End Sub
    End Class
End Namespace
